ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.html

```diff
@@ -1,136 +1,348 @@
+               role="tooltip"
+               class="form-control tooltip tooltip-validation tooltip-lg tooltip-top-left"
+               [class.invalid]="form.get('cpuLimit').invalid && (form.get('cpuLimit').dirty || form.get('cpuLimit').touched)">
+
+          <input id="cpu-limit-selector"
+                 class="form-control"
+                 type="text"
+                 formControlName="cpuLimit">
           <span class="tooltip-content">{{ formErrMessage }}</span>
-          <div class="maxResourceIndicator">Max: {{ resourceLimits.cpu.maxUsage || '-' }} MHz, Min: 1 MHz</div>
-        </label>&nbsp;MHz
-        </div>
-        <div class="form-group">
-          <label class="p5" for="memory-limit-selector">Memory</label>
-          <label class="p5" for="memory-limit-selector" aria-haspopup="true" role="tooltip" class="tooltip tooltip-validation tooltip-lg tooltip-top-left"
-            [class.invalid]="form.get('memoryLimit').invalid && (form.get('memoryLimit').dirty || form.get('memoryLimit').touched)">
-            <input id="memory-limit-selector" type="text" formControlName="memoryLimit">
-            <span class="tooltip-content">{{ formErrMessage }}</span>
-            <div class="maxResourceIndicator">Max: {{ resourceLimits.memory.maxUsage || '-' }} MB, Min: 1 MB</div>
-          </label>&nbsp;MB
-        </div>
+
+          <small class="text-muted">
+            Max: {{ resourceLimits.cpu.maxUsage || '-' }} MHz, Min: 1 MHz
+          </small>
+
+        </label>
+
       </div>
-      <a id="advanced-toggle" class="btn btn-link" (click)="toggleAdvancedMode()">{{ inAdvancedMode ? 'Basic' : 'Advanced' }} <clr-icon [attr.shape]="inAdvancedMode ? 'caret up' : 'caret down'"></clr-icon></a>
-      <div *ngIf="inAdvancedMode">
-        <div class="alert alert-sm alert-info mb10">
-          <div class="alert-item">
-            <span class="alert-text">
-              The following options will be used for VCH creation only when the advanced mode is open.
-            </span>
-          </div>
+
+      <span>MHz</span>
+
+    </div>
+
+    <div class="form-group row">
+      <div class="col-xs-4">
+
+        <label for="memory-limit-selector">
+          Memory
+        </label>
+
+      </div>
+      <div class="col-xs-3">
+
+        <label for="memory-limit-selector"
+               aria-haspopup="true"
+               role="tooltip"
+               class="form-control tooltip tooltip-validation tooltip-lg tooltip-top-left"
+               [class.invalid]="form.get('memoryLimit').invalid && (form.get('memoryLimit').dirty || form.get('memoryLimit').touched)">
+
+          <input id="memory-limit-selector"
+                 class="form-control"
+                 type="text"
+                 formControlName="memoryLimit">
+          <span class="tooltip-content">{{ formErrMessage }}</span>
+
+          <small class="text-muted">
+            Max: {{ resourceLimits.memory.maxUsage || '-' }} MB, Min: 1 MB
```

Comment:
The Min value shouldn't be defined in the markup it should be same as Max. I know it was already here but if you can update it that would be great.